app/services/aws/credentials_service.rb

```diff
@@ -8,46 +8,58 @@ def call(params = {})
       end
     end
 
-    attr_reader :params
+    AuthenticationTypeError = Class.new(StandardError)
+    TokenNotFoundError = Class.new(StandardError)
```

Comment:
https://rubystyle.guide/#single-line-classes

Reply:
Updated in f391f58.